tests/hint.test

```diff
@@ -239,3 +239,35 @@ Found:
 Note: Custom message
 
 4 hints
+
+---------------------------------------------------------------------
```

Comment:
Normally end to end tests like this are excessive for simple rules - see the `<TEST>` block in `data/hlint.yaml` which will run faster and be easier to write

Reply:
I'm still in the dark as to why the e2e tests were failing, but thanks for the suggestion. This is covered in 013928c01a63060946e7a254af93c928337acc24.